testpackage/tests/transtest_amr/transtest_amr.cfg

```diff
@@ -49,10 +49,11 @@ vy_min = -2.0e6
 vy_max = +2.0e6
 vz_min = -2.0e6
 vz_max = +2.0e6
-vx_length = 1
-vy_length = 1
-vz_length = 1
-max_refinement_level = 1
+vx_length = 30
```

Comment:
Is this supposed to be now so much bigger?

Reply:
Reverted this - although to note that transtest_amr is designed for testing velocity AMR, not spatial AMR.